### ISSUE TITLE
New changelog

### ISSUE THREAD

**Issue by PWagner1** [Wagnerp/Krypton-NET-5.470#50]
New HTML5 & CSS3 based changelog. Thoughts?

**Comment by Smurf-IV** [comment]
Do those styles work with GitHub - readme?
or do we have to put the changes in 2 places ?

**Comment by Wagnerp** [comment]
It's in pure HTML & CSS, unfortunately I don't know if it's compatible with the GitHub repo, since it doesn't render when clicked on. I'll be styling the table with CSS3 in due course, but I just wanted to make sure that the colours, design etc worked before making it more permanent.

**Comment by Smurf-IV** [comment]
Something I have found that might be useful for upgrade detection:
https://github.com/Smurf-IV/Elucidate/commits

and for new releases:
https://www.somsubhra.com/github-release-stats/?username=Smurf-IV&repository=Elucidate
or 
https://githubstats0.firebaseapp.com/

**Comment by Wagnerp** [comment]
What do you think of: https://wagnerp.github.io/Krypton-NET-5.470/?

**Comment by Smurf-IV** [comment]
Nice :-)

**Comment by Smurf-IV** [comment]
Is the need for the `Updates` directory removed, with the above Link ?
Also the Above link does not display the Icon:
![image](https://user-images.githubusercontent.com/2418812/55061244-27257600-506b-11e9-9550-f4cf12d0f504.png)

**Comment by Wagnerp** [comment]
It should be working off `README.md`. I don't know if the icon needs to be in the root, as at the moment, it's in the `Assets\PNG\Square Design` directory.

**Comment by Smurf-IV** [comment]
![image](https://user-images.githubusercontent.com/2418812/59147252-39b81600-89f1-11e9-9ec5-45809b130aeb.png)

**Comment by Wagnerp** [comment]
Yes, I don't know why this happens, as all relevant assets are in the root
